tests/config.h

```diff
@@ -88,6 +94,7 @@ typedef struct master_config {
    // Test-execution configuration parameters
    uint64 seed;
    uint64 num_inserts;
+   uint64 num_threads;
```

Comment:
This looks like it got replaced by the `num_normal_bg_threads` and `num_memtable_bg_threds` fields.  Delete this one.

Reply:
Check if `num_threads` is still needed; withdraw it if not.

Reply:
`num-threads` was added for a new test case I had developed, `large_inserts_stress_test`, but that test has now been withdrawn.

This config param was left behind. I've deleted it now.

And gently cleaned up some definitions in `config.c` for mnemonics related to bg-threads.

Reply:
I still see it?